test/TestAsset/Anonymous/with-anonymous-class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      https://github.com/zendframework/zend-file for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-file/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\File\TestAsset\Anonymous;
+
+$anonymous = new class extends \stdClass {
```

Comment:
To clarify: the intent of the class is to find files that declare _named_ classes. It was originally developed as a utility for creating class maps for autoloading. Anonymous classes, as they are not named, would fall outside that criteria.